### PR TITLE
core-svc-chore(logs): improve logging on create news module

### PR DESCRIPTION
**Pull request by rachadiannovansyah**
## Overview
- Write logging on create news module

## Changes
- Add logging on process create news
- Add logging on process login

## Evidence:
project: Portal Jabar
title: improve logging on create news & auth module
participants: @rachadiannovansyah @alqinsidev @ayocodingit

### CODE REVIEW

**Review by alqinsidev — APPROVED** [review]
Good, it straight forward enough

File: core-service/src/modules/news/delivery/http/news_handler.go [hunk 1]
```diff
@@ -165,13 +171,17 @@ func (h *NewsHandler) Store(c echo.Context) (err error) {
...
 	if err != nil {
+		h.Logger.Error(log, err)
 		return err
 	}
 
 	// Copy slice to slice
 	res := []domain.DetailNewsResponse{}
 	copier.Copy(&res, &n)
 
+	log.AdditionalInfo["news_created"] = &n.CreatedBy.Name
```
Comment (ayocodingit) [inline core-service/src/modules/news/delivery/http/news_handler.go:182]: I think we need to opd name? Wdyt

Comment (rachadiannovansyah) [reply core-service/src/modules/news/delivery/http/news_handler.go:182]: `&n.CreatedBy.Name` is the representation of opd name @ayocodingit

**Review by ayocodingit — APPROVED** [review]
Looks good

**Pull request merged**